homeassistant/components/autarco/strings.json

```diff
@@ -23,6 +23,30 @@
   },
   "entity": {
     "sensor": {
+      "flow_now": {
+        "name": "Flow now"
+      },
```

Comment:
What does flow now mean?

Reply:
It shows the battery flow in watts. I called it flow because there is current (watts) flowing to and from the battery.

Reply:
How would one use this? Is it always positive? Or is it positive when charging and negative while decharging?

Reply:
Positive when the battery is charging and negative when it is discharging.